Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -766,6 +766,51 @@ public void test_checkWrongCardsDataInSets() {
         }
     }
 
+    @Test
+    @Ignore
+    public void test_checkWrongFullArtAndRetro() {
```

Comment:
Must clean up original `test_checkWrongCardsDataInSets` from same checks with retro art and other.